src/Root.cpp

```diff
@@ -159,6 +159,14 @@ void cRoot::Start(std::unique_ptr<cSettingsRepositoryInterface> a_OverridesRepo)
 	m_MojangAPI = new cMojangAPI;
 	bool ShouldAuthenticate = settingsRepo->GetValueSetB("Authentication", "Authenticate", true);
 	m_MojangAPI->Start(*settingsRepo, ShouldAuthenticate);  // Mojang API needs to be started before plugins, so that plugins may use it for DB upgrades on server init
+	AString MaxLoadedChunks = settingsRepo->GetValueSet("Server", "MaxChunkRAM", "30");
+	m_MinimizeRam = settingsRepo->GetValueSetB("Server", "MinimizeRAM", true);
+	if (!StringToInteger(MaxLoadedChunks, m_MaxRAMChunks))
+	{
+		settingsRepo->SetValue("Server", "MaxChunkRAM", "30");
+		m_MaxRAMChunks = 30;  // in MiB
```

Comment:
This is global for the entire server. It is not a hard limit - If the server needs more, it can use more, but it would "fight to lower it back" to 30MiB and the strategy would be effectively the "agressive" strategy mentioned at the first post.